lang/french.json

```diff
@@ -236,19 +256,174 @@
+    "sydneyhud_hudlist_show_minion_count" : "Compteur de Joker",
+    "sydneyhud_hudlist_show_minion_count_desc" : "Montre un compteur pour le nombre d'ennemis convertis sur la carte",
+    "sydneyhud_hudlist_show_pager_count" : "Compteur de pagers",
+    "sydneyhud_hudlist_show_pager_count_desc" : "Montre un compteur pour le nombre de pagers utilisés. Peut être incorrect si rejoignant en pleine partie. Désactivé automatiquement quand le braquage passe en mode bourrin",
+    "sydneyhud_hudlist_show_camera_count" : "Compteur de caméras",
+    "sydneyhud_hudlist_show_camera_count_desc" : "Montre un compteur pour le nombre de caméra actives sur la carte. N'inclus pas les caméras drone. Désactivé automatiquement quand le braquage passe en mode bourrin",
+    "sydneyhud_hudlist_show_body_count" : "Compteur de corps",
+    "sydneyhud_hudlist_show_body_count_desc" : "Montre un compteur de corps morts sur la carte. Désactivé automatiquement quand le braquage passe en mode bourrin",
+    "sydneyhud_hudlist_show_loot" : "Compteur de butin",
+    "sydneyhud_hudlist_show_loot_desc" : "Montre un compteur pour le nombre et les types d'items de butin sur la carte. Peut ne pas afficher le butin si pas encore activé ou apparu. L'option globale rassemble toutes les types en un seul item",
+    "sydneyhud_hudlist_separate_bagged_loot" : "Sacs de butin séparé",
+    "sydneyhud_hudlist_separate_bagged_loot_desc" : "Montre des valeurs différentes pour les sacs de butin, et ceux qui ne le sont pas",
+    "sydneyhud_hudlist_show_special_pickups" : "Pickup spéciaux",
+    "sydneyhud_hudlist_show_special_pickups_desc" : "Décocher pour désactiver l'affichage d'équipement/pickup spécial sur la carte (Si activé, les types individuels peuvent être activés/désactivés dans le sous menu",
+    "sydneyhud_hudlist_show_buffs" : "Montre les Buffs",
+    "sydneyhud_hudlist_show_buffs_desc" : "Décocher pour désactiver l'affichage de tout les buffs (Si activé, les buffs individuels peuvent être activés/désactivés ci-dessous)",
+    "sydneyhud_hudlist_show_player_actions" : "Montrer les actions du joueur",
+    "sydneyhud_hudlist_show_player_actions_desc" : "Décocher pour désactiver l'affichage de toutes les actions effectuées (Si activé, les actions individuelles peuvent être activés/désactivés ci-dessous)",
+    "sydneyhud_hudlist_not_ignore_item_crowbar" : "Pieds de biche",
+    "sydneyhud_hudlist_not_ignore_item_courier" : "Paquet de coursier de Gages",
+    "sydneyhud_hudlist_not_ignore_item_keycard" : "Carte-clé",
+    "sydneyhud_hudlist_not_ignore_item_meth_ingredients" : "Ingrédients de Meth",
+    "sydneyhud_hudlist_not_ignore_item_planks" : "Planches",
+    "sydneyhud_hudlist_not_ignore_item_secret_item" : "Items secrets",
+    "sydneyhud_hudlist_not_ignore_item_secret_item_desc" : "Différents items spéciaux tel que les bagues dans Birth of Sky et les bouteilles sur le Biker Heist",
```

Comment:
Birth of Sky in French is "Genèse Celéste"
Biker Heist in French is "À Pleinz Gaz"

Reply:
Forgot that the map names were localized. It's been a while, and I've immediately only remembered how I was habitually using the original names, and so, forgetting to check for the original localized ones.